src/DDTrace/Integrations/Nette/NetteIntegration.php

```diff
@@ -0,0 +1,176 @@
+    {
+        return false;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function init()
+    {
+        if (!self::shouldLoad(self::NAME)) {
+            return self::NOT_AVAILABLE;
+        }
+
+        if (\PHP_MAJOR_VERSION < 7) {
+            $integration = $this;
+            \DDTrace\hook_method('Nette\Configurator', '__construct', function () use ($integration) {
+                $integration->load();
+            });
+        } else {
+            $this->load();
+        }
+
+        return self::LOADED;
+    }
+
+    public function load()
+    {
+        $scope = GlobalTracer::get()->getRootScope();
+        if (!$scope instanceof Scope) {
+            return;
+        }
+
+        $service = \ddtrace_config_app_name(NetteIntegration::NAME);
+
+        $root = $scope->getSpan();
+        $root->setTraceAnalyticsCandidate();
+        $root->overwriteOperationName('nette.request');
+        $root->setTag(Tag::SERVICE_NAME, $service);
+
+        \DDTrace\trace_method(
+            'Nette\Configurator',
+            'createContainer',
+            function (SpanData $span) use ($service) {
+                $span->name = 'nette.configurator.createContainer';
+                $span->type = Type::WEB_SERVLET;
+                $span->service = $service;
+            }
+        );
+
+        \DDTrace\trace_method(
+            'Nette\Configurator',
+            'createRobotLoader',
+            function (SpanData $span) use ($service) {
+                $span->name = 'nette.configurator.createRobotLoader';
+                $span->type = Type::WEB_SERVLET;
+                $span->service = $service;
+            }
+        );
+
+        \DDTrace\trace_method(
+            'Nette\Application\Application',
+            'run',
+            function (SpanData $span) use ($root, $service) {
+                $span->name = 'nette.application.run';
+                $span->type = Type::WEB_SERVLET;
+                $span->service = $service;
+                $root->setTag(Tag::HTTP_STATUS_CODE, http_response_code());
+            }
+        );
+
+        \DDTrace\trace_method(
+            'Nette\Application\UI\Presenter',
+            'run',
+            function (SpanData $span, $args) use ($root, $service) {
+
+                $span->name = 'nette.presenter.run';
+                $span->type = Type::WEB_SERVLET;
+                $span->service = $service;
+
+                if (count($args) < 1 || !\is_a($args[0], '\Nette\Application\Request')) {
+                    return;
+                }
+
+                $request = $args[0];
+                $presenter = $request->getPresenterName();
+                $action = $request->getParameter('action');
+
+                $root->setTag(Tag::HTTP_METHOD, $request->getMethod());
+                $root->setTag('nette.route.presenter', $presenter);
+                $root->setTag('nette.route.action', $action);
+                $root->setResource($presenter . ':' . $action);
```

Comment:
This is how we used to set the root span resource names, but since the route data is already being stored in the span metadata, the root span resource name will default to the HTTP method + URL endpoint. So we can remove this.

Reply:
Done